src/http-ama-server.tsx

```diff
@@ -0,0 +1,78 @@
+import { renderToReadableStream } from "react-dom/server";
+
+import { setupOllama } from './setup-ollama.ts';
+// import { check } from './check.ts';
+
+let PORT = 3000;
+
+let clientEventSourceScript = `
+  let eventSource = new globalThis.EventSource('/ollama-status');
+  let statusElement = document.getElementById('ollamaStatus')
+  statusElement.textContent = 'sourcing...'
+
+  eventSource.addEventListener('status', (event) => {
+    
+    let statusElement = document.getElementById('ollamaStatus')
+    console.log(statusElement)
+
+    if (statusElement) {
+      statusElement.textContent = event.data
+      eventSource.close()
```

Comment:
close the event source only when the status is one of `success` or `already_available`